keyboards/kprepublic/bm40hsrgb/keymaps/simchee/rules.mk

```
@@ -0,0 +1,11 @@
+BOOTLOADER = atmel-dfu
+MOUSEKEY_ENABLE = yes # Mouse keys
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+UNICODE_ENABLE = yes         # Unicode
+EXTRAKEY_ENABLE = yes # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
```

You only need this, really: 
```suggestion
UNICODE_ENABLE = yes         # Unicode
```